tests/indent/tsx_spec.lua

```diff
@@ -0,0 +1,26 @@
+local Runner = require("tests.indent.common").Runner
+local runner = Runner:new(it, "tests/indent/tsx", {
+  tabstop = 2,
+  shiftwidth = 2,
+  expandtab = true,
+})
+
+describe("indent JSX Elements:", function()
+  describe("whole file:", function()
+    runner:whole_file(".", {
+      expected_failures = {},
+    })
+  end)
+
+  describe("new line:", function()
+    runner:new_line("issue-3986.tsx", { on_line = 5, text = "<div></div>", indent = 8 })
+    runner:new_line("issue-3986.tsx", { on_line = 6, text = "<div></div>", indent = 6 })
+    runner:new_line("issue-3986.tsx", { on_line = 7, text = "<div></div>", indent = 6 })
+    runner:new_line("issue-3986.tsx", { on_line = 8, text = "<div></div>", indent = 4 })
+    runner:new_line("element_attributes.tsx", { on_line = 4, text = "className={'testClass'}", indent = 8 })
+    runner:new_line("element_attributes.tsx", { on_line = 6, text = "backgroundColor: 'red'", indent = 10 })
+    runner:new_line("element_attributes.tsx", { on_line = 8, text = "}", indent = 8 })
```

Comment:
You might use the approach of ecma_spec.lua/javascript_spec.lua that combines those repeated function calls

Reply:
Done